data/content_scripts/posting_process/privly_app.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @fileOverview Script acting as a "middleman" to allow communication between
+ * the extension and privly application.
+ */
+
+/**
+ * Handles the receipt of initial content from the extension. This content is then 
+ * sent to the privly application
+ * 
+ * @param {Object} data Message sent by the extension. This includes the initial content 
+ *                      and the message secret(used to establish communication between the
+ *                      content scripts and privly-applications).
+ */
+function initialContentHandler(data) {
+  var message = JSON.stringify({
+                  secret: data.secret,
+                  handler: "initialContent",
+                  initialContent: data.initialContent,
+                });
+  window.postMessage(message, "*");
```

Comment:
Yup, this content script is injected only in [privly-applications](https://github.com/hitesh96db/privly-jetpack/blob/tests/lib/posting_process.js#L118).